ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -250,6 +250,27 @@
     reason="Test runs ONLY on OSD or ROSA cluster",
 )
 
+multicluster_platform_required = pytest.mark.skipif(
```

Comment:
wouldn't multicluster_platform_required imply also DR setup?
How about we rename these 2 marks to:

provider_client_platform_required
provider_client_ms_platform_required